README.md

```diff
@@ -161,6 +161,34 @@ will be added as they are required.
 
 ---
 
+`should_not_redirect`
+
+  + Use this method when you need to validate that a given request is not redirected
+
+**Usage:**
+
+  should\_not\_redirect TEST\_CASE\_NUMBER SOURCE\_PARAMETERIZED\_URL 
+
+**Example:**
+
+	should_not_redirect 0001 http://${targetSite}/original/location
+
+---
+
+`should_rewrite`
```

Comment:
Yep, I'm all on on this one.  I'll merge it in.  

BTW, when making multiple changes to a codeline with the intent on performing a pull request, it's much easier for the code maintainer to accept individual features if each feature has it's own commit.  In this fashion, I could accept the addition of `should_rewrite` even when I have questions on `should_not_redirect`.  When all the changes are in one big super-commit, it's an all or nothing accept.

The only way I can accept the parts of the pull request that I agree with and don't have remaining questions on is to make the changes myself, which I can do, but I'd prefer you get credit for the work you did and have your name set as the owner of the code change.